src/containers/consumerOnboarding/components/sandbox/validateForm.ts

```diff
@@ -37,6 +43,23 @@ export const validateForm = (values: Values): FormikErrors<Values> => {
     errors.oAuthRedirectURI = validateOAuthRedirectURI(values.oAuthRedirectURI);
   }
 
+  if (anyInternalApisSelected(values.apis)) {
+    const vaEmailPattern = /^[A-Za-z0-9._%+-]+@va.gov$/;
+
+    errors.internalApiInfo = {
+      programName: validatePresence('program name', values.internalApiInfo.programName),
+      sponsorEmail: validateEmail(values.internalApiInfo.sponsorEmail),
+      // eslint-disable-next-line no-negated-condition
+      vaEmail: !vaEmailPattern.test(values.email) ? validateVAEmail(values.internalApiInfo.vaEmail) : '',
```

Comment:
I actually prefer it as you have it, but I think this is a valid option to remove the eslint comment
```suggestion
      vaEmail: vaEmailPattern.test(values.email) ? '' : validateVAEmail(values.internalApiInfo.vaEmail),
```

Reply:
tried it the other was and it prevents the form from submitting